src/main/java/com/ing/data/cassandra/jdbc/Utils.java

```diff
@@ -135,6 +135,7 @@
     public static final String TAG_CLOUD_SECURE_CONNECT_BUNDLE = "secureConnectBundle";
     public static final String TAG_CONFIG_FILE = "configFile";
     public static final String TAG_REQUEST_TIMEOUT = "requestTimeout";
+    public static final String TAG_COMPLIANCE_MODE = "complianceMode";
```

Comment:
You also have to create a cosntant "KEY_COMPLIANCE_MODE" (see above in the code) with a full lower case name to take it into account in the JDBC URL as query parameter, then map it to the corresponding tag after the line 309 in this file (method `parseURL`). This is probably not very optimized but refactoring this part, inherited from the original project, is not currently the priority.